ui/app/components/app/permissions-connect-header/permissions-connect-header.component.js

```diff
@@ -5,13 +5,14 @@ import IconWithFallBack from '../../ui/icon-with-fallback'
 export default class PermissionsConnectHeader extends Component {
   static propTypes = {
     icon: PropTypes.string,
-    iconName: PropTypes.string.isRequired,
+    iconName: PropTypes.string,
     headerTitle: PropTypes.node,
     headerText: PropTypes.string,
   }
 
   static defaultProps = {
     icon: null,
+    iconName: 'Unknown External Domain',
```

Comment:
There's a hot second where this prop becomes undefined because the permissions request is deleted, if the request is approved in another UI instance.

Reply:
Is this still a problem? We've solved this in other places by caching the relevant prop in state, so we can show it after deletion. But it'd be nice to avoid it altogether if possible.

Showing nothing might be better than this. "Unknown External Domain" isn't the most helpful thing to show, and it's also not localized. Showing nothing could be accomplished using `<Redirect to={path}>` in place of `history.push` - the `Redirect` component navigates and renders nothing.

Reply:
This was just throwing an error, it was never really visible. I changed it to an empty string and that should be fine.

Reply:
d8e7fb4